guides/source/command_line.md

```diff
@@ -102,8 +102,8 @@ Please choose a generator below.
 
 Rails:
   assets
+  channel
   controller
-  generator
```

Comment:
@rafaelfranca I thought it would be better not to change the number of lines. But I noticed it's not make sense, and restored it. Thanks.